src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+
+if (!tree) {
+  throw new Error('List not defined');
```

Comment:
The error message 'List not defined' might be misleading. Consider changing it to something more descriptive, like 'Tree element not found', to better reflect the actual issue.